storage/storage.go

```diff
@@ -4,23 +4,29 @@ import (
 	"context"
 	"io"
 
-	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/specs-actors/actors/abi"
 )
 
 type Data = io.Reader
 
 type Storage interface {
 	// Creates a new empty sector
-	NewSector(ctx context.Context) (abi.SectorNumber, error)
+	NewSector(ctx context.Context, sector abi.SectorID) error
```

Comment:
This is the responsibility of FSM / market connector to fsm

Reply:
ok, leave that in the method description.